### ISSUE TITLE
About the pretrained weight

### ISSUE THREAD

**Issue by kevin655** [Alibaba-MIIL/ASL#29]
Hi, I'm trying to implement your novel loss function in my dataset. 
One thing about your ASL paper is confusing me, that is, which of the following pretrained model you used to implement ASL loss in your experiment?
https://github.com/Alibaba-MIIL/TResNet/blob/master/MODEL_ZOO.md
1. tresnet_l.pth
2. tresnet_l_448.pth

Thanks in advance!
Best

**Comment by mrT23** [comment]
the models you refer to are pretained on imagenet.
tresnet_l is pretrained on resolution 224
tresnet_l_448 is pretrained on resolution 448
ASL is not dependent on them. you can try any other architecture with it.
in the paper we show results on 3 different architectures.
![image](https://user-images.githubusercontent.com/21198860/102634300-4ecf4280-415a-11eb-9b96-7c0c67d39607.png)

specifically for your question, we used tresnet_l

**Comment by kevin655** [comment]
Thank you for your reply, I get what you mean.

Best
